cypress/page/login.page.ts

```diff
@@ -0,0 +1,29 @@
+class LoginPage {
+    private emailselector: string;
+    private passwordselector: string;
+    private email: string;
+    private password: string;
+    private submitlog: string
+
+    constructor() {
+        this.emailselector = '#email'
+        this.email = 'aperdomobo@gmail.com'
+        this.passwordselector = '#passwd'
+        this.password = 'WorkshopProtractor'
+        this.submitlog = '#SubmitLogin > span'
+    }
+
+    public typeEmail(): void {
+        cy.get(this.emailselector).type(this.email)
```

Comment:
the email should be an input param of the function, or input param of the constructor, it should not be hardcoded on this class :)